events/events.go

```diff
@@ -78,7 +78,7 @@ func (p *publisher) Get() interface{} {
 }
 
 func (p *publisher) GetAndSubscribe() (interface{}, Subscriber, error) {
-	c := make(chan struct{})
+	c := make(chan struct{}, 2)
```

Comment:
I thought the channel is only used for notifying the subscribers and doesn't have the actual data. If so, it should be fine if the notification is dropped before the subscriber has a chance to read it since all the subscriber needs to know is just there's an update and it should go read it?